certified-connectors/Azure Communication Services Email/apiDefinition.swagger.json

```diff
@@ -0,0 +1,301 @@
+                },
+                "attachments": {
+                  "type": "array",
+                  "title": "Attachments",
+                  "description": "List of attachments.",
+                  "x-ms-visibility": "advanced",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "title": "Name",
+                        "description": "Attachment name with extension.",
+                        "x-ms-visibility": "important"
+                      },
+                      "attachmentType": {
+                        "type": "string",
+                        "title": "Type",
+                        "description": "The type of attachment file.",
+                        "x-ms-visibility": "internal"
+                      },
+                      "contentBytesBase64": {
+                        "format": "byte",
+                        "type": "string",
+                        "title": "Content",
+                        "description": "Attachment content.",
+                        "x-ms-visibility": "important"
+                      }
+                    },
+                    "required": [
+                      "name",
+                      "attachmentType",
+                      "contentBytesBase64"
+                    ]
+                  }
+                },
+                "importance": {
+                  "type": "string",
+                  "title": "Importance",
+                  "description": "The importance type for the email.",
+                  "enum": [
+                    "high",
+                    "normal",
+                    "low"
+                  ],
+                  "default": "normal",
+                  "x-ms-visibility": "advanced"
+                },
+                "disableUserEngagementTracking": {
+                  "type": "boolean",
+                  "title": "Disable User Engagement Tracking",
+                  "description": "Indicates whether user engagement tracking should be disabled for this request if the resource-level user engagement tracking setting was already enabled in the control plane.",
+                  "x-ms-visibility": "advanced"
+                }
+              },
+              "required": [
+                "sender",
+                "recipients",
+                "content"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error.",
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code - this will be the same as the code in the error property in the response body.",
+                "type": "string"
+              }
+            }
+          },
+          "202": {
+            "description": "Accepted.",
+            "schema": {},
```

Comment:
@lei-msft empty response schemas are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).